sos/plugins/insights.py

```diff
@@ -13,16 +13,16 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['insights-client']
```

Comment:
Can you be **sure** that all systems that will run the updated plugin will also have the updated Insights package? If not, then we should leave both possible names in the plugin for now, until the old name is entirely obsolete.

Reply:
The new insights-client package obsolete the older package redhat-access-insights so ideally every client should have new/updated package installed. 

# rpm -q insights-client --obsoletes 
redhat-access-insights <= 1.0.13-2.el7_5
redhat-access-proactive <= 0.3.3-0.el7_5

In this case, should we keep both package names?